common/common/src/main/java/dev/enola/common/io/resource/TestResource.java

```diff
@@ -19,45 +19,52 @@
 
 import com.google.common.net.MediaType;
 
-import java.io.Closeable;
 import java.io.IOException;
 import java.net.URI;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicLong;
 
-public class MemoryResourcePool {
+public class TestResource extends MemoryResource implements CloseableResource {
 
-    private static final Map<Long, CloseableMemoryResource> pool = new HashMap<>();
+    private static final String SCHEME = "memory";
```

Comment:
I'll change this in a follow-up PR:

```suggestion
    private static final String SCHEME = "test";
```